bin/build-release.sh

```diff
@@ -67,24 +71,37 @@ function package {(
 
   local tarName="storm-mesos-${RELEASE}.tgz"
   cd _release
-  tar czf ${tarName} apache-storm-${RELEASE}
+  # When supervisor starts up it looks for storm-mesos not apache-storm.
+  mv apache-storm-${RELEASE} storm-mesos-${RELEASE}
```

Comment:
Doesn't this conflict with your other PR? That's already merged, so you need to rebase